shard-operator/src/main/resources/application.properties

```diff
@@ -31,4 +31,6 @@ event-bridge.webhook.account-id=${WEBHOOK_TECHNICAL_ACCOUNT_ID:402cf429-da04-493
 %openshift.quarkus.kubernetes.deployment-target=openshift
 %openshift.quarkus.container-image.build=true
 %openshift.event-bridge.k8s.platform=openshift
-event-bridge.shard-id=${EVENT_BRIDGE_SHARD_ID:150b23e9-0b34-45b7-a91a-5ec388d03a1d}
\ No newline at end of file
+event-bridge.shard-id=${EVENT_BRIDGE_SHARD_ID:150b23e9-0b34-45b7-a91a-5ec388d03a1d}
+
+quarkus.log.category."com.redhat.service.bridge.shard.operator".level=DEBUG
```

Comment:
Is this a debugging leftover?

Reply:
It wasn't as a lot of our _useful_ logging is at `debug` level and we made a conscious decision to enable it for `manager`. This log level is only for our package but if it is causing too much noise I can revert.

Reply:
fine for me to keep it, was just surprised by the change